src/gen-indic-table.py

```diff
@@ -83,6 +79,390 @@
+  0x0954: 'SM',
+
+  # The following act like consonants.
+  0x0A72: 'C',
+  0x0A73: 'C',
+  0x1CF5: 'C',
+  0x1CF6: 'C',
+
+  # TODO: The following should only be allowed after a Visarga.
+  # For now, just treat them like regular tone marks.
+  0x1CE2: 'A',
+  0x1CE3: 'A',
+  0x1CE4: 'A',
+  0x1CE5: 'A',
+  0x1CE6: 'A',
+  0x1CE7: 'A',
+  0x1CE8: 'A',
+
+  # TODO: The following should only be allowed after some of
+  # the nasalization marks, maybe only for U+1CE9..U+1CF1.
+  # For now, just treat them like tone marks.
+  0x1CED: 'A',
+
+  # The following take marks in standalone clusters, similar to Avagraha.
+  0xA8F2: 'Symbol',
+  0xA8F3: 'Symbol',
+  0xA8F4: 'Symbol',
+  0xA8F5: 'Symbol',
+  0xA8F6: 'Symbol',
+  0xA8F7: 'Symbol',
+  0x1CE9: 'Symbol',
+  0x1CEA: 'Symbol',
+  0x1CEB: 'Symbol',
+  0x1CEC: 'Symbol',
+  0x1CEE: 'Symbol',
+  0x1CEF: 'Symbol',
+  0x1CF0: 'Symbol',
+  0x1CF1: 'Symbol',
+
+  0x0A51: 'M', # https://github.com/harfbuzz/harfbuzz/issues/524
+
+  # According to ScriptExtensions.txt, these Grantha marks may also be used in Tamil,
+  # so the Indic shaper needs to know their categories.
+  0x11301: 'SM',
+  0x11302: 'SM',
+  0x11303: 'SM',
+  0x1133B: 'N',
+  0x1133C: 'N',
+
+  0x0AFB: 'N', # https://github.com/harfbuzz/harfbuzz/issues/552
+  0x0B55: 'N', # https://github.com/harfbuzz/harfbuzz/issues/2849
+
+  0x0980: 'PLACEHOLDER', # https://github.com/harfbuzz/harfbuzz/issues/538
+  0x09FC: 'PLACEHOLDER', # https://github.com/harfbuzz/harfbuzz/pull/1613
+  0x0C80: 'PLACEHOLDER', # https://github.com/harfbuzz/harfbuzz/pull/623
+  0x0D04: 'PLACEHOLDER', # https://github.com/harfbuzz/harfbuzz/pull/3511
+
+  0x2010: 'PLACEHOLDER',
+  0x2011: 'PLACEHOLDER',
+
+  0x25CC: 'DOTTEDCIRCLE',
+
+
+  # Khmer
+
+  0x179A: 'Ra',
+
+  0x17CC: 'Robatic',
+  0x17C9: 'Robatic',
+  0x17CA: 'Robatic',
+
+  0x17C6: 'Xgroup',
+  0x17CB: 'Xgroup',
+  0x17CD: 'Xgroup',
+  0x17CE: 'Xgroup',
+  0x17CF: 'Xgroup',
+  0x17D0: 'Xgroup',
+  0x17D1: 'Xgroup',
+
+  0x17C7: 'Ygroup',
+  0x17C8: 'Ygroup',
+  0x17DD: 'Ygroup',
+  0x17D3: 'Ygroup', # Just guessing. Uniscribe doesn't categorize it.
+
+
+  # Myanmar
+
+  # https://docs.microsoft.com/en-us/typography/script-development/myanmar#analyze
+
+  0xFE00: 'VS',
+  0xFE01: 'VS',
+  0xFE02: 'VS',
+  0xFE03: 'VS',
+  0xFE04: 'VS',
+  0xFE05: 'VS',
+  0xFE06: 'VS',
+  0xFE07: 'VS',
+  0xFE08: 'VS',
+  0xFE09: 'VS',
+  0xFE0A: 'VS',
+  0xFE0B: 'VS',
+  0xFE0C: 'VS',
+  0xFE0D: 'VS',
+  0xFE0E: 'VS',
+  0xFE0F: 'VS',
+
+  0x104E: 'C', # The spec says C, IndicSyllableCategory doesn't have.
+
+  0x002D: 'PLACEHOLDER',
+  0x00A0: 'PLACEHOLDER',
+  0x00D7: 'PLACEHOLDER',
+  0x2012: 'PLACEHOLDER',
+  0x2013: 'PLACEHOLDER',
+  0x2014: 'PLACEHOLDER',
+  0x2015: 'PLACEHOLDER',
+  0x2022: 'PLACEHOLDER',
+  0x25FB: 'PLACEHOLDER',
+  0x25FC: 'PLACEHOLDER',
+  0x25FD: 'PLACEHOLDER',
+  0x25FE: 'PLACEHOLDER',
+
+  0x1004: 'Ra',
+  0x101B: 'Ra',
+  0x105A: 'Ra',
+
+  0x1032: 'A',
+  0x1036: 'A',
+
+  0x1039: 'H',
```

Comment:
Change Indic_Syllabic_Category=Invisible_Stacker to map to `H` and remove this override.